packages/sources/ncfx/src/endpoint/crypto.ts

```diff
@@ -13,27 +16,17 @@ import { transport } from '../transport/crypto'
 
 export const inputParameters = new InputParameters(priceEndpointInputParametersDefinition)
 
-type Response = {
-  Result: number
-  bid: number
-  ask: number
-  Data: {
-    result: number
-    bid: number
-    mid: number
-    ask: number
-  }
-}
+type OmitResultFromLwba = Omit<LwbaResponseDataFields, 'Result'>
 
 export type BaseEndpointTypes = {
   Parameters: typeof inputParameters.definition
-  Response: Response
+  Response: OmitResultFromLwba & SingleNumberResultResponse
   Settings: typeof config.settings
 }
 
 export const cryptoEndpoint = new CryptoPriceEndpoint({
```

Comment:
Why can't this use `LwbaEndpoint`?

Reply:
NCFX (and also the future GSR) lwba endpoints are just aliases of the default crypto price endpoint.
They could be changed to be LWBA endpoints (with `crypto` as an alias), but IMO better to leave it this way and just enforce the response types.